client/cypress/integration/pages/flightupload.spec.js

```diff
@@ -24,8 +26,8 @@ describe("check admin page", () => {
     cy.loginNormalUser();
 
     cy.get("button").contains("Flug hochladen").click();
-
-    cy.get("h3").should("have.text", `Flug hochladen`);
+    // TODO: This fails…
```

Comment:
Grundsätzlich sagt Cypress als Hinweis: One or more matched elements are not visible.
Er findet leider mehrere h3

Siehe auch: https://docs.cypress.io/guides/references/best-practices#Selecting-Elements

Habe auch schon überlegt. Sowas wie .page-title und .dialog-title einzuführen.